decidim-initiatives/spec/lib/decidim/initiatives/engine_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Initiatives::Engine do
+  it "loads engine and app previews" do
+    expect(ActionMailer::Preview.all).to match_array([
+                                                       Decidim::DeviseMailerPreview,
+                                                       Decidim::Initiatives::InitiativesMailerPreview
+                                                     ])
```

Comment:
```suggestion
      expect(ActionMailer::Preview.all).to include(Decidim::Initiatives::InitiativesMailerPreview)
```

Let's change it to only checking the relevant class

Reply:
Done 😄